WooCommerce/src/main/kotlin/com/woocommerce/android/ui/compose/component/Toolbar.kt

```diff
@@ -44,40 +47,42 @@ fun ToolbarWithHelpButton(
     )
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun Toolbar(
     modifier: Modifier = Modifier,
     title: String = "",
     onNavigationButtonClick: (() -> Unit),
     navigationIcon: ImageVector = Icons.AutoMirrored.Filled.ArrowBack,
     navigationIconContentDescription: String = stringResource(id = string.back),
-    windowInsets: WindowInsets = WindowInsets(0),
+    windowInsets: WindowInsets = TopAppBarDefaults.windowInsets,
 ) {
     Toolbar(
         modifier = modifier,
-        title = { Text(title) },
+        title = { Text(title, style = MaterialTheme.typography.headlineSmall) },
```

Comment:
```suggestion
        title = { Text(title) },
```

The `headlineSmall` style is causing a significant change from the old implementation, and thus causing inconsistencies with other screens:

| Before | After |
| --- | --- |
| <img width="1080" height="2400" alt="Screenshot_20250731_145419" src="https://github.com/user-attachments/assets/613a484a-8da9-4da1-920d-d51879313d37" /> | <img width="1080" height="2400" alt="Screenshot_20250731_145317" src="https://github.com/user-attachments/assets/383a9df7-6fe1-423a-853f-9374e20e4d2c" /> |

I tested removing the explicit style to use the default style provided by the `TopAppBar`, and it seems to be closer to what had before:

<img width="360" alt="Screenshot_20250731_145728" src="https://github.com/user-attachments/assets/ac986060-fa80-4e6d-9681-d95bd276ea05" />

Reply:
Good catch!

51a9bb178df2bf9870f7273d7b1e2e25f46744cb